### DISCUSSION TITLE
what is error?

### DISCUSSION THREAD

**Discussion by asgharb** [jasontaylordev/CleanArchitecture#606] [General]
An unhandled exception occurred while processing the request.
SocketException: No connection could be made because the target machine actively refused it.
System.Net.Sockets.Socket+AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)

HttpRequestException: No connection could be made because the target machine actively refused it. (localhost:4200)
System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(string host, int port, HttpRequestMessage initialRequest, bool async, CancellationToken cancellationToken)

HttpRequestException: Failed to proxy the request to http://localhost:4200/, because the request to the proxy target failed. Check that the proxy target server is running and accepting requests to http://localhost:4200/.

The underlying exception message was 'No connection could be made because the target machine actively refused it. (localhost:4200)'.Check the InnerException for more details.

**Comment by bramburn** [comment]
follow this to get it working
The easiest way to get started is to install the [NuGet package](https://www.nuget.org/packages/Clean.Architecture.Solution.Template) and run dotnet new ca-sln:

    Install the latest [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
    Install the latest [Node.js LTS](https://nodejs.org/en/)
    Run dotnet new --install Clean.Architecture.Solution.Template to install the project template
    Create a folder for your solution and cd into it (the template will use it as project name)
    Run dotnet new ca-sln to create a new project
    Navigate to src/WebUI/ClientApp and run npm install
    Navigate to src/WebUI/ClientApp and run npm start to launch the front end (Angular)
    Navigate to src/WebUI and run dotnet run to launch the back end (ASP.NET Core Web API)
